src/main/java/com/ibm/disni/rdma/verbs/impl/RdmaVerbsNat.java

```diff
@@ -275,26 +231,25 @@ public int destroyCQ(IbvCQ cq) throws IOException {
 	//--------------------------------------
 
 	void free(NatRegMrCall natRegMrCall) {
-		this.regList.add(natRegMrCall);
 	}
 
 	public void free(NatDeregMrCall natDeregMrCall) {
-		this.deregList.add(natDeregMrCall);
+
 	}
 
 	public void free(NatPostSendCall natPostSendCall) {
-		this.postSendList.add(natPostSendCall);
+
 	}
 
 	public void free(NatPostRecvCall natPostRecvCall) {
-		this.postRecvList.add(natPostRecvCall);
+
 	}
 
 	public void free(NatPollCqCall natPollCqCall) {
-		this.pollCqList.add(natPollCqCall);
+
 	}
 
 	public void free(NatReqNotifyCall natReqNotifyCall) {
-		this.reqNotifyList.add(natReqNotifyCall);
+
```

Comment:
The PR looks good now, except that I think we can remove all those free() methods now that they are not doing anything, no?

Reply:
Done

Reply:
@patrickstuedi please review